velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -899,6 +899,57 @@ PlanBuilder& PlanBuilder::localMerge(const std::vector<std::string>& keys) {
   return *this;
 }
 
+PlanBuilder& PlanBuilder::expand(
+    const std::vector<std::vector<std::string>>& projections) {
+  std::vector<std::vector<core::TypedExprPtr>> projectExprs;
+  projectExprs.reserve(projections.size());
+  const auto numColumns = projections[0].size();
+  const auto numRows = projections.size();
+  std::vector<std::string> names;
+  names.reserve(numColumns);
+  std::vector<TypePtr> types;
+  types.reserve(numColumns);
+  int grouIdColCount = 0;
+  const auto inputRowType = planNode_->outputType();
+
+  for (auto i = 0; i < numRows; i++) {
+    std::vector<core::TypedExprPtr> projectExpr;
+    for (auto j = 0; j < numColumns; j++) {
+      auto untypeExpression = parse::parseExpr(projections[i][j], options_);
+      auto typeExpression = inferTypes(untypeExpression);
+
+      if (names.size() != numColumns) {
+        if (auto fieldExpr = dynamic_cast<const core::FieldAccessExpr*>(
+                untypeExpression.get())) {
+          names.push_back(fieldExpr->getFieldName());
```

Comment:
fieldExpr can also have an alias, no? "c0 as foo"

Reply:
@mbasmanova Maybe not. The input type doesn't contain the foo name.

Reply:
Please, try "c0 as foo" projection. You'll find that fieldExpr->alias() returns foo.